### DISCUSSION TITLE
Full-Text Search

### DISCUSSION THREAD

**Discussion by DhivakarK-git** [isar/isar#711] [Help]
How would u search results for a sentence?

Split it and query for each word?

So use a associated where clauses with query modifier?

**Comment by muddassar-lab** [comment]
Isar.dev contains documentation on how to use full text search.Here is the link
https://isar.dev/recipes/full_text_search.html#basic-example

**Comment by tobzilla90** [comment]
Hi, 
I am experiencing trouble with the provided example in the docs.
For me the **Any** queries are not showing up (...**AnyEqualTo**, ...**AnyStartsWith**)
The **EqualTo** and **NoEqualTo** queries are showing but they won't help me in terms of full text search.
Has anyone experience the same issue?

Here is how I am building the Index:
`  
@Index(type: IndexType.value, caseSensitive: false)
 List<String> get searchContent => stringValueField.split(' ');
`

And when I want to write the Query:
![Screen Shot 2022-12-07 at 20 00 13](https://user-images.githubusercontent.com/23441397/206162397-43fa9efb-2b60-4cf1-a415-3ee4f1e89a04.png)

Using Isar version 3.0.5

**Comment by kaankoken** [comment]
As far as I see, `AnyEqualTo` does not exist.

**Reply by codykl** [reply]
If you look at the changelog [here](https://github.com/isar/isar/blob/a6f7cea6b0a99c1c1fd289ab569dda3e6b6780bb/packages/isar/CHANGELOG.md). There is a line that says:

Renamed multi-entry where clauses from .yourListAnyEqualTo() to .yourListElementEqualTo() to avoid confusion
